docs/index.md

```diff
@@ -208,7 +208,7 @@ Every {py:class}`~pytket.backends.backend.Backend` in pytket has its own {py:met
 * - [AutoRebase [2]](inv:#*.AutoRebase)
   - [SynthesiseTket](inv:#*.SynthesiseTket)
   - [FullPeepholeOptimise](inv:#*.passes.FullPeepholeOptimise)
-  - [RemoveBarriers](inv:#*pytket.passes.RemoveBarriers)
+  - [RemoveBarriers](inv:#*pytket._tket.passes.RemoveBarriers)
```

Comment:
This is needed as there is a `RemoveBarriers` class in qiskit as well so sphinx complains that the target is not unique. Adding the `_tket` makes the link unambiguous. It is replaced by the build script so no `_tket` links appear in the built pages